### PR TITLE
Math magicians: components

### PR DESCRIPTION
**Pull request by kingqabil**
- Used React components.

- Used React props.

- Used React state.

### CODE REVIEW

File: src/components/calculator.js [hunk 1]
```diff
@@ -0,0 +1,113 @@
+import { Component } from 'react';
+import './calculator.css';
+
+/* eslint-disable react/prefer-stateless-function */
+class Calculator extends Component {
```
Comment (TedLivist) [inline src/components/calculator.js]: - [ ] Linters help us write clean code. Let us maintain that. Kindly enable linters here.

  **Suggestion**: For the linter error, as suggested in the linter message, you could set up your state within a constructor, before the `render` method, like so:

    ```
      constructor(props) {
          super(props);
          this.state = {};
      }
    ```

File: src/App.js [hunk 2]
```diff
@@ -1,29 +1,12 @@
...
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+/* eslint-disable react/prefer-stateless-function */
+class App extends Component {
```
Comment (TedLivist) [inline src/App.js]: - [ ] Kindly enable linters here and resolve errors accordingly

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @kingqabil,

Good job so far!
✔️ Nice UI
✔️ Good documentation
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- [ ] Users cannot really run your app locally without installing packages with `npm install`. Kindly add that command as a necessary instruction in your README.
 
- [ ] According to the requirements you are supposed to have deleted CRA boilerplate files. Kindly delete the following files in the **public** directory: `logo192.png`, `logo512.png`, `manifest.json`, `robots.txt`. Also delete the following files in the **src** directory: `App.test.js`, `logo.svg`.
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ZeenatLawal — APPROVED** [review]
Hi @kingqabil, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

## Congratulations! 🎉 
6vEo


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**